script/test

```diff
@@ -25,138 +25,151 @@
+# shellcheck source=script/common.sh
+. "$(dirname "$0")"/common.sh
 
 set -e
 
 OT_DIR=${OT_DIR:-$HOME/src/openthread}
 if [[ $(uname) == "Darwin" ]]; then
-  OTBIN_DIR=$OT_DIR/output/x86_64-apple-darwin/bin
+    OTBIN_DIR=$OT_DIR/output/x86_64-apple-darwin/bin
 else
-  OTBIN_DIR=$OT_DIR/output/x86_64-unknown-linux-gnu/bin
+    OTBIN_DIR=$OT_DIR/output/x86_64-unknown-linux-gnu/bin
 fi
 
-installed() {
-  command -v $1 >/dev/null 2>&1
+installed()
+{
+    command -v "$1" >/dev/null 2>&1
 }
 
-go_tests() {
-  install_deps
-  install
+go_tests()
+{
+    install_deps
+    install_otns
 
-  echo "" >coverage.txt
+    echo "" >coverage.txt
 
-  PKG=$(go list ./...)
+    PKG=$(go list ./...)
 
-  for d in $PKG; do
-    go test -race -coverprofile=profile.out -covermode=atomic $d
-    if [ -f profile.out ]; then
-      cat profile.out >>coverage.txt
-      rm profile.out
-    fi
-  done
+    for d in $PKG; do
+        go test -race -coverprofile=profile.out -covermode=atomic "$d"
+        if [ -f profile.out ]; then
+            cat profile.out >>coverage.txt
+            rm profile.out
+        fi
+    done
 }
 
-install_deps() {
-  ./script/install-deps
+install_deps()
+{
+    ./script/install-deps
 }
 
-install() {
-  ./script/install
+install_otns()
+{
+    ./script/install
 }
 
-get_openthread() {
-  if ! [[ -x $OT_DIR ]]; then
-    mkdir -p "$(dirname $OT_DIR)"
-    git clone https://github.com/openthread/openthread.git $OT_DIR
-    cd $OT_DIR
-    ./script/bootstrap
-    ./bootstrap
-    cd -
-  fi
+get_openthread()
+{
+    if ! [[ -x $OT_DIR ]]; then
+        mkdir -p "$(dirname "$OT_DIR")"
+        git clone https://github.com/openthread/openthread.git "$OT_DIR"
+        (
+            cd "$OT_DIR"
+            ./script/bootstrap
+            ./bootstrap
+        )
+    fi
 }
 
-build_openthread() {
-  get_openthread
-  cd $OT_DIR
-  make -f $OT_DIR/examples/Makefile-simulation OTNS=1
-  cd -
+build_openthread()
+{
+    get_openthread
+    cd "$OT_DIR"
+    make -f "$OT_DIR"/examples/Makefile-simulation OTNS=1
+    cd -
 }
 
-py_unittests() {
-  install_deps
-  install
-  build_openthread
-  cd $OTBIN_DIR
-  python3 $OTNSDIR/pylibs/unittests/test_basic.py
-  python3 $OTNSDIR/pylibs/unittests/test_ping.py
-  python3 $OTNSDIR/pylibs/unittests/test_commissioning.py
-  cd -
+py_unittests()
+{
+    install_deps
+    install_otns
+    build_openthread
+    cd "$OTBIN_DIR"
+    python3 "$OTNSDIR"/pylibs/unittests/test_basic.py
+    python3 "$OTNSDIR"/pylibs/unittests/test_ping.py
+    python3 "$OTNSDIR"/pylibs/unittests/test_commissioning.py
+    cd -
 }
 
-check_py_example() {
-  echo "Checking pyOTNS example: $1 ..."
-  python3 $1 &
-  local pid=$!
-  sleep 10
-  killall otns || true
-  wait $pid || die "Run pyOTNS example failed :$1"
+check_py_example()
+{
+    echo "Checking pyOTNS example: $1 ..."
+    python3 "$1" &
+    local pid=$!
+    sleep 10
+    killall otns || true
+    wait $pid || die "Run pyOTNS example failed :$1"
```

Comment:
```suggestion
    wait "$pid" || die "Run pyOTNS example failed :$1"
```
Do we always quota variables?

Reply:
Not sure. `shellchecking` is not complaining here, so it should be just fine.